includes/helper/class-wp-job-manager-helper.php

```diff
@@ -545,39 +545,41 @@ function ( $plugin ) use ( $search_term ) {
 	/**
 	 * Return if we should show or not the bulk activation form.
 	 *
-	 * @param array $licenced_plugins The list of licensed plugins to handle.
+	 * @param array $licenced_plugins The list of licenced plugins to handle.
 	 *
 	 * @return bool If we should show the bulk activation form or not.
 	 */
 	private function show_bulk_activation_form( $licenced_plugins ) {
 		foreach ( array_keys( $licenced_plugins ) as $product_slug ) {
 			$licence = self::get_plugin_licence( $product_slug );
-			if ( empty( $licence['licence_key'] ) && apply_filters( 'wpjm_display_license_form_for_addon', true, $product_slug ) ) {
+			if ( empty( $licence['licence_key'] ) && apply_filters( 'wpjm_display_licence_form_for_addon', true, $product_slug ) ) {
 				return true;
 			}
 		}
 		return false;
 	}
 
 	/**
-	 * Outputs unset license key notices.
+	 * Outputs unset licence key notices.
 	 */
-	public function licence_error_notices() {
```

Comment:
@aaronfc and I had discussed renaming, so it's just a rename that's right.

I've not deprecated a method before, but that sounds like a plan to me. I'll get to looking at that.

Reply:
Deprecated here https://github.com/Automattic/WP-Job-Manager/pull/2409/commits/ed89a13d0a2f23d86fb65b376bfef5dae6322a28

Reply:
Thank you for changing that, Katja! And sorry for the late delay on reviewing it!

Just instead of `maybe_add_license_error_notices()` we should use `WP_Job_Manager_Helper::maybe_add_license_error_notices` to reference the correct method. Not something critical and urgent though. 😉